klarna-on-demand/src/main/java/com/klarna/ondemand/UrlHelper.java

```diff
@@ -50,12 +50,8 @@ static String preferencesUrl(String token) {
                 .appendPath(token)
                 .appendPath("preferences")
                 .appendQueryParameter("api_key", Context.getApiKey())
-                .appendQueryParameter("locale", defaultLocale())
+                .appendQueryParameter("locale", Context.getLocale(context))
```

Comment:
I'm sure you can tell me what I'm missing, but this construct (`Context.getLocale(context)`) is quite awkward. Is there no way around this?